package.json

```diff
@@ -14,7 +14,7 @@
     "notebook:win": "npm run build && docker pull docker-hub.x-lab.info/opendigger/open-digger-jupyter-nodejs:1.0 && docker run -it --rm -p 8888:8888 -v %cd%:/home/node/notebook docker-hub.x-lab.info/opendigger/open-digger-jupyter-nodejs:1.0",
     "pull-label-file-test": "tsc && node lib/ci/pull_label_file_test.js",
     "cron": "npm run build && node --max-old-space-size=8192 lib/cron/index.js",
-    "clean": "rm -rf dist"
+    "test": "CLICKHOUSE_HOST=http://106.15.39.211:8123 CLICKHOUSE_USERNAME=default mocha"
```

Comment:
It means that the test clickhouse server can be accessed by anyone?

Reply:
Yes, if we want to test SQLs on PR and push automatically, the service needs to be public.

Although I think I should create a readonly account for CI test rather than use default account which has write permission.

Reply:
> Although I think I should create a readonly account for CI test rather than use default account which has write permission.

sounds great.